models/intermediate/int_zendesk__organization_aggregates.sql

```diff
@@ -1,8 +1,10 @@
+{{ config(enabled=var('using_organizations', True)) }}
+
 with organizations as (
     select * 
     from {{ ref('stg_zendesk__organization') }}
 
---If you use organization tags this will be included, if not it will be ignored.
+--If you use organization tags and organizations, this will be included, if not it will be ignored.
```

Comment:
Do we need "and organizations"? Since that's presumed based on the model config on line 1.

Reply:
Ah yes, that should have been removed as well.